tests/e2e/testRunner.js

```diff
@@ -202,6 +203,12 @@ const runTests = async () => {
             }
         }
 
+        const coolDownLogs = Logger.progressInfo(`Cooling down for ${config.COOL_DOWN / 1000}s`);
+        coolDownLogs.updateText(`Cooling down for ${config.COOL_DOWN / 1000}s`);
+
+        await sleep(config.BOOT_COOL_DOWN);
```

Comment:
Having the cooldown right at the beginning should hopefully lower the chances of heat throttling from the previous run (which we have no control over and will be a completely different AWS DF customer/app). It also gives the time to cool down between test suites.